scss/variables.scss

```diff
@@ -0,0 +1,132 @@
+// =============================================
+// Twitter Bootstrap variables
+// =============================================
+$font-size-base: 0.8rem !default;
+$font-family-sans-serif: 'Source Sans Pro', sans-serif !default;
+
+$line-height-computed: 20px !default;
+$input-default-height: 30px !default;
+
+$headings-font-weight: 300 !default;
+
+$border-radius:        2px !default;
+$border-radius-lg:       2px !default;
+$border-radius-sm:       2px !default;
+
+// =============================================
+// Twitter boostrap Colors
+// =============================================
+$state-warning-text: #af7f18 !default;
+$state-warning-bg: floralwhite !default;
+$state-warning-border: #fcf8e3 !default;
+
+$state-danger-text: #b13d31 !default;
+$state-danger-bg: #fdefee !default;
+$state-danger-border: #fad7db !default;
+
+$state-success-text: #229173 !default;
+$state-success-bg: #f2fcf9 !default;
+$state-success-border: #ddf7eb !default;
+
+$state-info-text: #26929c !default;
+$state-info-bg: #eef8fc !default;
+$state-info-border: #d0f2f7 !default;
+
+$grid-gutter-width:         0 !default;
+
+// =============================================
+// MODALS
+// =============================================
+$zindexModalBackdrop: 1040 !default;
+$zindexModal: 1050 !default;
+
+// =============================================
+// PROGRESS BARS
+// =============================================
+$baseBorderRadius: 3px !default;
+
+// =============================================
+// TOOLTIPS
+// =============================================
+$font-size-mini: ceil($font-size-base * 0.85) !default;
+// ~11px
+
+$tooltip-color: #fff !default;
+$tooltip-bg: rgba(0, 0, 0, .9) !default;
+$tooltip-arrow-width: 5px !default;
+$tooltip-arrow-color: $tooltip-bg !default;
+
+$zindex-tooltip: 1030 !default;
+
+
+// =============================================
+// BASE FONT SIZE AND FAMILY
+// =============================================
+$baseTextColor: #666 !default;
+$baseFontWeight: 400 !default;
+$boldFontWeight: 600 !default;
+$line-height-base: 20px !default;
+
+// =============================================
+// SECTIONS
+// =============================================
+
+$applicationHeaderHeight: 40px !default;
+$applicationHeaderPadding: 15px !default;
+$applicationHeaderFontSize: 1.2em !default;
+$sidebarWidth: 340px !default;
+$sidebarNavItemWidth: 40px !default;
+$sidebarResizerWidth: 6px !default;
+$sidebarResizerColor: #e1e1e1 !default;
+$sidebarResizerActiveColor: #ccc !default;
```

Comment:
Name of variable `sidebarResizerActiveColor` should be written in all lowercase letters with hyphens instead of underscores